pages/pipelines/hosted_agents/terminal_access.md

```diff
@@ -0,0 +1,32 @@
+# Hosted agents terminal access
+
+The Buildkite hosted agents feature provides you with _terminal/console access_ to jobs running on hosted agents. This feature is useful in allowing you to:
+
+- Understand what components are installed, as you set up your pipeline.
+- Test the behavior of different scripts (because they may not be well-documented).
+- Debug issues that are not reproducible in your local environment.
+
+This can be useful when migrating your pipelines across to [queues](/docs/clusters/manage-queues) on Buildkite hosted agents.
+
+## Activate terminal access on hosted agents
+
+Before you can use the terminal access feature on your Buildkite hosted agents, you'll need to activate this feature for your Buildkite organization, which requires Buildkite organization administrator permissions.
```

Comment:
We've changed the default. The feature is enabled for all organizations, but can be disabled within the Organization Settings.

Reply:
Thanks, I'll repurpose/rewrite this section to be 'deactivate and reactivate terminal access on hosted agents'.

Reply:
This has now been done and since this feature is now active by default, I've moved this section to the end of the page.